src/wp-includes/update.php

```diff
@@ -522,15 +539,16 @@ function wp_update_themes( $extra_stats = array() ) {
 	$translations     = wp_get_installed_translations( 'themes' );
 
 	$last_update = get_site_transient( 'update_themes' );
+
 	if ( ! is_object( $last_update ) ) {
-		$last_update = new stdClass;
+		$last_update = new stdClass();
 	}
 
 	$themes  = array();
 	$checked = array();
 	$request = array();
 
-	// Put slug of current theme into request.
+	// Put slug of active theme into request.
 	$request['active'] = get_option( 'stylesheet' );
 
 	foreach ( $installed_themes as $theme ) {
```

Comment:
Seems that from [changeset 53933 Themes: Add support for Update `URI header`](https://core.trac.wordpress.org/changeset/53933) it's missing:
```php
'UpdateURI'  => $theme->get( 'UpdateURI' ),
```
Maybe other part of the changeset are missing, but this one was triggering a warning:
`Undefined array key "UpdateURI" in /.../ClassicPress-v2/src/wp-includes/update.php on line 691`

Reply:
I have not done much yet with the update code as it risks breaking updated of core. but this addition seems reasonable to add in with a few more differences in this file.